### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by hendridg**
## In this Milestone 3:

- Insert the contact `<form>` in the `<footer>`
- Make sure that:
  Name field accepts up to 30 chars.
  Email field accepts only emails.
  Message field accepts up to 500 chars.
  All fields have placeholder text.
  All fields have the correct type.
  A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
 
>I working alone, my coding partner withdraw of Microverse

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -232,3 +233,77 @@ li {
...
+  border: 2px solid #008552;
+}
+
+input:invalid,
+textarea:invalid {
+  border: 2px solid red;
+}
+
+input:invalid:required,
+textarea:invalid:required {
```
Comment (lily-coder) [inline style.css]: - [ ] [REQUIRED] Please ensure you stick to the design templates in figma as much as possible, this includes the font, colours, images and spacing (margin/padding). You can change the background color in the input form to match the original
## YOUR DESIGN
Capture11
## ORIGINAL DESIGN
Capture12

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @hendridg , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good commit messages
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (hendridg) [reply style.css]: Thanks, @lily-coder,

I thought I could adjust some to my liking, thanks for the information, now I will strictly abide by the design

**Review by lily-coder — APPROVED** [review]
Hi @hendridg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hendridg** [comment]
Thanks @lily-coder

**Pull request merged**